projects/angular-calendar/src/modules/common/calendar-a11y.pipe.ts

```diff
@@ -19,7 +19,10 @@ export class CalendarA11yPipe implements PipeTransform {
   ) {}
 
   transform(a11yParams: A11yParams, method: string): string {
+    console.log('-----------------');
+    console.log(a11yParams.locale);
     a11yParams.locale = a11yParams.locale || this.locale;
+    console.log(a11yParams.locale);
```

Comment:
These console logs are causing the Travis CI build to be flooded with logs. They should be removed